scipy/signal/_peak_finding.py

```diff
@@ -257,24 +257,39 @@ def peak_prominences(vector, peaks, wlen=None):
         Indices of peaks in `vector`.
     wlen : int, optional
         A window lenght in samples that limits the search for the lowest
-        contour line two a symmetric interval around the evaluated peak. If
-        not given the entire vector is used. Use this parameter to speed up
-        the calculation significantly for large vectors.
+        contour line two a symmetric interval around the evaluated peak. If not
```

Comment:
sentence is not complete I guess, "for the lowest contour line two a symmetric interval around [...]" ?

Reply:
Ah, that's a typo. The sentence should be "[...] for the lowest contour line to a symmetric interval around the evaluated peak."